test/server.js

```diff
@@ -175,3 +175,97 @@ test('test get parties by type and id, mcc, mnc invalid', async t => {
     t.is(response.statusCode, 200);
     t.deepEqual(JSON.parse(response.payload), expectedResult);
 });
+
+test('test put parties by type and id', async t => {
```

Comment:
why do we call this `put parties` instead of `put participants`?

Reply:
Dumb mistake.